index.js

```diff
@@ -45,8 +45,16 @@ const debug = createDebugMessages('license-report');
     // Get a list of all the dependencies we want information about.
     const inclusions = util.isNullOrUndefined(config.only) ? null : config.only.split(',')
     const exclusions = Array.isArray(config.exclude) ? config.exclude : [config.exclude]
+    let exclusionRegexp
+    if ((config.excludeRegex !== undefined)  && (typeof config.excludeRegex === 'string') && (config.excludeRegex !== '')) {
+      try {
+        exclusionRegexp = new RegExp(config.excludeRegex, 'i')
+      } catch (error) {
+        exclusionRegexp = undefined
```

Comment:
perhaps an error should be logged and the program stopped if the regex is faulty?

Reply:
I agree concerning the logging of the error - we should at least tell the user that something went wrong and not silently ignore that fact. I simply missed this point.

Concerning what to do besides logging, I am not so sure.
* IMHO the program should be terminated, if it makes no sense to ignore the error and simply go on with the rest of the program. An example would be, if the package,json to inspect does not exist,
* But in my opinion when we get a useful output even when ignoring the error, we should ignore it (after logging it) and go on with the rest of the program. In our case, this will result in more packages to be returned than expected, but at least we have a list of packages.

What do you think?